spec/supabase_js_v2.yml

```diff
@@ -583,7 +659,14 @@ functions:
           ```js
           supabase.auth.exchangeCodeForSession('34e770dd-9ff9-416c-87fa-43b31d7ef225')
           ```
+  - id: auth-mfa-api
```

Comment:
@J0 btw i added a new section for the MFA methods so it's now nested under a subheader